src/pages/displayAllRepositories/index.tsx

```diff
@@ -0,0 +1,68 @@
+import './App.css';
+import { useContext, useState, useEffect } from 'react';
+import { getCommitsByRepository, getRepositoriesByUser } from '../../service';
+import { NavBar } from '../../components/navbar';
+import { UserProfileContext } from '../../context/user';
+import { Card } from '../../components/card';
+import { IRepositoryProps } from '../../types/repositories';
+
+
+export const DisplayAllRepositories = () => {
+
+  const [repositories, setRepositories] = useState<IRepositoryProps[]>([]);
+  const [nickname, setNickName] = useState('');
+  const { user, isStorage, getStorageProfileUser, getProfileUserStored } = useContext(UserProfileContext);
```

Comment:
I didn't understand the variables `getStorageProfileUser` and `getProfileUserStored`... I recommend to change the name of them to something more specific